docker/util.sh

```diff
@@ -5,6 +5,42 @@
 #
 # Utility library to use in scripts
 
+# This function installs packages with specific versions and also pin them
+# (with prio 990) to those versions by appending pinning information to
+# /etc/apt/preferences.d/cachalot, so they are not upgraded accidentally.
+#
+# All packages should be specified with version: pkg=version.
+#
+# Version can have wildcards, see apt-get/apt_preferences documentation for
+# details
+apt_pin_install()
+{
+	( { set +x -e; } 2>/dev/null # dissable verboseness (and silently)
```

Comment:
`(and silently)`? also typo `dissable`

Reply:
"Disables verboseness silently". You need a trick not to get a `+set +x` printed if `set -x` was set before :)

Reply:
Ah, I got it now.

Reply:
Added commit to fix the typos (there was the same type in other parts of this file)

Reply:
Maybe I could remove the **and** (`# disable verboseness (silently)`) as part of the typo fixing commit. Do you think is more clear that way?

Reply:
I think it's clearer that way, but I don't think it makes much difference.

Reply:
Done.